nav/virtual_layers/src/virtual_layer.cpp

```diff
@@ -0,0 +1,146 @@
+#include <virtual_layers/virtual_layer.h>
+#include <pluginlib/class_list_macros.h>
+
+#include <random>
+
+PLUGINLIB_EXPORT_CLASS(virtual_layers::VirtualLayer, costmap_2d::Layer)
+
+using costmap_2d::LETHAL_OBSTACLE;
+using costmap_2d::NO_INFORMATION;
+
+namespace virtual_layers
+{
+
+VirtualLayer::VirtualLayer() {
+  // ros::Time last_updated_ = ros::Time::now();
```

Comment:
Remove commented out code

Reply:
Also it might be better to change move ros::Time last_updated_ iniialization to the onInitialize function instead of in the header